server/services/context.py

```diff
@@ -264,11 +265,35 @@ def _try_recover_context_from_exception(self, start_offset: int) -> None:
         self._try_get_attribute_context_at_line_position(target_offset)
 
     def _try_get_tag_context_at_line_position(self, target_offset):
-        tag_match = re.search(START_TAG_REGEX, self._context.document_line, re.DOTALL)
-        if tag_match and tag_match.start(TAG_GROUP) <= target_offset <= tag_match.end(TAG_GROUP):
-            self._context.token_type = ContextTokenType.TAG
-            self._context.token_name = tag_match.group(TAG_GROUP)
-            raise ContextFoundException()
+        tag_matches = re.finditer(START_TAG_REGEX, self._context.document_line, re.DOTALL)
+        if tag_matches:
+            for matchNum, match in enumerate(tag_matches, start=1):
```

Comment:
matchNum is not used and can be removed togeter with enumerate?

Reply:
Well spotted! Leftovers from the regex investigation :)

Reply:
I'm actually surprised that linting has not caught that.

Reply:
Yes, normally it cries, I am also surprised